Packs/DeveloperTools/Integrations/XsoarServerPowershell/XsoarServerPowershell.ps1

```diff
@@ -0,0 +1,90 @@
+. $PSScriptRoot\CommonServerPowerShell.ps1
+
+$script:INTEGRATION_NAME = "XSOAR Server Powershell"
+$script:COMMAND_PREFIX = "xsoar"
+$script:INTEGRATION_ENTRY_CONTEX = "XSOAR"
+
+#### HELPER FUNCTIONS ####
+
+function SetIntegrationContextCommand() {
```

Comment:
It looks like you defined this function twice. See line 34.

Reply:
Its a mistake, It should be `TestModuleCommand` changed.